data_structures/hashing/number_theory/chinese_remainder_theorem.py

```diff
@@ -0,0 +1,70 @@
+# Chinese Remainder Theorem:
+
+# If GCD(a,b) = 1, then for any remainder ra modulo a and any remainder rb modulo b there exists integer n,
+# such that n = ra (mod a) and n = ra(mod b).  If n1 and n2 are two such integers, then n1=n2(mod ab)
+
+# Algorithm :
+
+# 1. Use extended euclid algorithm to find x,y such that a*x + b*y = 1
+# 2. Take n = ra*by + rb*ax
+
+# import testmod for testing our function
+from doctest import testmod
```

Comment:
Let's move this line down just below line 68...
We do not need this import when we run your functions, only when we test them.